packages/datadog-instrumentations/index.js

```diff
@@ -8,3 +8,4 @@ require('./src/when')
 require('./src/promise')
 require('./src/q')
 require('./src/promise-js')
+require('./src/couchbase')
```

Comment:
It's technically out of scope of this PR, but can you reorder these alphabetically? It makes the list easier to inspect.

Reply:
It also helps with merge conflicts later on.